docs/sdkx-xamarin/outbound-support.mdx

```diff
@@ -139,57 +137,72 @@ public override void OnMessageReceived(RemoteMessage remoteMessage)
+                var resultIntent = new Intent(this, typeof(ProactiveNotificationHandlerActivity));
+
+                // Pass the values to ProactiveNotificationHandlerActivity:
+                resultIntent.PutExtras(valuesForActivity);
+                resultIntent.SetFlags(ActivityFlags.NewTask);
+
+                PendingIntentFlags flag = Build.VERSION.SdkInt < BuildVersionCodes.M ? 0 : PendingIntentFlags.Immutable;
+
+                // Create the PendingIntent and update current if there is already a pendingIntent created earlier.
+                var resultPendingIntent = PendingIntent.GetActivity(this, 0, resultIntent, flag | PendingIntentFlags.UpdateCurrent);
+
+                // Build the notification:
+                var builder = new NotificationCompat.Builder(this, MyApplication.CHANNEL_ID)
+                              .SetAutoCancel(true)
+                              .SetContentIntent(resultPendingIntent) // Start up this activity when the user clicks the intent.
+                              .SetContentTitle("Proactive Notification") // Set the title
+                              .SetSmallIcon(HelpshiftApiExample.Droid.Resource.Drawable.app_icon) // This is the icon to display
+                              .SetContentText("Click to see proactive in action!"); // the message to display.
+
+                // Finally, publish the notification:
+                var notificationManager = NotificationManagerCompat.From(this);
+                notificationManager.Notify(1, builder.Build());
+         }
     }
 }
 ```
 
-When the user clicks on the notification, `HandleNotificationActivity` activity is started (as specified in the above code sample).
-To delegate push notification data to Helpshift SDK, call `Helpshift.HandleProactiveLink()` in **HandleNotificationActivity's onCreate()** method.
+When the user clicks on the notification, `ProactiveNotificationHandlerActivity` activity is started (as specified in the above code sample).
+To delegate push notification data to Helpshift SDK, call `Helpshift.HandleProactiveLink()` in **ProactiveNotificationHandlerActivity's onCreate()** method.
 
 For Example -
 
-_Following code shows how to delegate push notification data by calling `Helpshift.handleProactiveLink()` to Helpshift SDK._
+_Following code shows how to delegate push notification data by calling `Helpshift.HandleProactiveLink()` to Helpshift SDK._
 
 ```csharp
-[Activity(Label = "HandleNotificationActivity")]
-public class HandleNotificationActivity : Activity
+[Activity(Label = "ProactiveNotificationHandlerActivity", MainLauncher = false)]
+public class ProactiveNotificationHandlerActivity : Activity
 {
     protected override void OnCreate(Bundle savedInstanceState)
     {
         base.OnCreate(savedInstanceState);
 
-        Bundle bundle = Intent.Extras;
-        if (bundle != null && bundle.GetBoolean("proactiveNotification"))
+        protected override void OnCreate(Bundle savedInstanceState)
```

Comment:
The OnCreate method is nested inside another OnCreate.

Reply:
fixed